src/main/java/com/dreammaster/mixin/mixins/early/MixinMinecraft_PackIcon.java

```diff
@@ -1,5 +1,9 @@
 package com.dreammaster.mixin.mixins.early;
 
+import java.net.URL;
+
+import javax.swing.*;
```

Comment:
why isn't spotless doing it for me smh

Reply:
I had to configure IntelliJ to stop doing it, too. 😆

Reply:
Spotless cannot do it and never has really. For it to be able to do it. It would need to be able to load up the entire project and see every class. Its currently a limitation that we can go around by making it delete wildcard imports and just stop the compiler that way. There is a setting in IntelliJ idea to prevent wildcard imports from being done by it automatically.

Reply:
Intellij has a nice "disambugate imports" option.  There is also a "auto wildcard import" that you can disable by setting it to 999